gnd/src/main/java/com/google/android/gnd/persistence/local/LocalValueStore.java

```diff
@@ -54,4 +54,12 @@ public void setLastActiveProjectId(@NonNull String id) {
   public void clearLastActiveProjectId() {
     preferences.edit().remove(ACTIVE_PROJECT_ID_KEY).apply();
   }
+
+  public boolean shouldUploadMediaOverUnmeteredConnectionOnly() {
```

Comment:
Nit: This change isn't related to checkstyle. Might be better to rebase master rather than merge in future.